app/gui/src/controller/searcher.rs

```diff
@@ -477,6 +477,7 @@ pub struct Searcher {
     ide:              controller::Ide,
     this_arg:         Rc<Option<ThisNode>>,
     position_in_code: Immutable<TextLocation>,
+    project_name:     Immutable<project::QualifiedName>,
```

Comment:
Is there a deeper reason why the searcher no stores the project name instead of deferring to the project? Does this account for renaming of the project?

Reply:
Please talk with @farmaazon about proper design here. Probably after project is initialized, we should have a data structure like `InitializedProject` and searcher should have access to it. This structure should contain all project-related information for initialized project. I believe there is more of info than just name.

Reply:
Yeah, I think a got a little too fancy here, we can store `model::Project`, and it is actually available in searcher's constructor.

Reply:
Refactored to storing `model::Project`